src/main/java/io/jenkins/plugins/audit/listeners/UserCreationListener.java

```diff
@@ -0,0 +1,86 @@
+package io.jenkins.plugins.audit.listeners;
+
+
+import hudson.Extension;
+
+import hudson.ExtensionList;
+import hudson.Extension;
+import io.jenkins.plugins.audit.RequestContext;
+import jenkins.security.SecurityListener;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.time.Instant;
+
+import io.jenkins.plugins.audit.event.CreateUser;
+import org.acegisecurity.userdetails.User;
+import org.apache.logging.log4j.audit.LogEventFactory;
+import org.acegisecurity.userdetails.UserDetails;
+
+/**
+ * Listener notified of user creation events.
+ */
+@Extension
+public class UserCreationListener extends SecurityListener {
+
+    /**
+     * Fired when a user was successfully authenticated using credentials. It could be password or any other credentials.
+     * This might be via the web UI, or via REST (using API token or Basic), or CLI (remoting, auth, ssh)
+     * or any other way plugins can propose.
+     *
+     * @param details details of the newly authenticated user, such as name and groups.
+     */
+    protected void authenticated(@Nonnull UserDetails details) {}
```

Comment:
You don't need to override the unused methods since the parent class has already defined them as empty blocks already.

Reply:
oh okay. will refactor `UserLogListener` as well, since it had similar.